odoo/addons/test_lint/tests/_odoo_checker_sql_injection.py

```diff
@@ -34,12 +34,30 @@ def _get_cursor_name(self, node):
         cursor_name = '.'.join(expr_list)
         return cursor_name
 
+    def _is_psycopg2_sql(self, node):
+        if isinstance(node, astroid.Name):
+            for assignation_node in self._get_assignation_nodes(node):
+                if self._is_psycopg2_sql(assignation_node):
+                    return True
+        if (not isinstance(node, astroid.Call) or
+                not isinstance(node.func, (astroid.Attribute, astroid.Name))):
+            return False
+        infered = checkers.utils.safe_infer(node)
+        # The package 'psycopg2' must be installed
+        if infered and infered.pytype().startswith('psycopg2'):
+            return True
+
     def _allowable(self, node):
         """
         :type node: NodeNG
         """
+        # sql.SQL or sql.Identifier is OK
```

Comment:
After looking at the CI failure, constants should probably be allowed as parameters.